configs/empirical_attention/faster_rcnn_r50_fpn_attention_0010_1x_coco.py

```diff
@@ -1,7 +1,13 @@
 _base_ = '../faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py'
 model = dict(
     backbone=dict(
-        gen_attention=dict(
-            spatial_range=-1, num_heads=8, attention_type='0010', kv_stride=2),
-        stage_with_gen_attention=[[], [], [0, 1, 2, 3, 4, 5], [0, 1, 2]],
-    ))
+        plugins=dict(conv2=[
```

Comment:
I just come up with another idea of plugins. It might not be better than the current implementation but I think it is worth consideration. It can be something like this:
```python
plugins=[
    dict(cfg=dict(
        type='GeneralizedAttention',
        spatial_range=-1,
        num_heads=8,
        attention_type='0010',
        kv_stride=2),
        stages=(False, True, True, True),
        position='after_conv2'),
    dict(cfg=dict(
        type='XXXXY',
        spatial_range=-1,
        num_heads=8,
        attention_type='0010',
        kv_stride=2),
        stages=(False, False, True, True),
        position='after_conv3'),
]
```
Then we can use something like `conv2_modules`, `conv3_modules` to add them in.
This seems to reduce the hierarchy.

Reply:
Sounds good to me.